### PR TITLE
release-notes-2.1-2.3

### PR DESCRIPTION
**Pull request by cedrabner**
Added release notes for linea v0.2.1-0.2-3

### CODE REVIEW

File: docs/developers/linea-version/release-notes.mdx [hunk 1]
```diff
@@ -17,7 +17,52 @@ Find out all you need to know about the latest Linea versions.
...
 > | Network | Version | Deployment Date |
 > | --- | --- | --- |
-> | Linea Goerli | [Alpha v0.2.0](#linea-alpha-v020-june-13-2023) | June 13, 2020 |
+> | Linea Goerli | [Alpha v0.2.3](#linea-alpha-v023-june-13-22-23) | June 13, 2020 |
+
+## Linea Alpha v0.2.3 (June 22, 2023)
+
+### Summary
+
+This release focuses on optimizing the provers hashing strategy to increase its computational efficiency.
```
Comment (thedarkjester) [inline docs/developers/linea-version/release-notes.mdx:26]: prover's

File: docs/developers/linea-version/release-notes.mdx [hunk 2]
```diff
@@ -17,7 +17,52 @@ Find out all you need to know about the latest Linea versions.
...
+
+## Linea Alpha v0.2.2 (June 20, 2023)
+
+### Summary
+
+This release focuses on updating the Postman Sdk configuration values and revising gas fee calculations.
```
Comment (thedarkjester) [inline docs/developers/linea-version/release-notes.mdx:32]: SDK

File: docs/developers/linea-version/release-notes.mdx [hunk 3]
```diff
@@ -17,7 +17,52 @@ Find out all you need to know about the latest Linea versions.
...
+
+### Features
+
+- Updated the Postman Sdk configuration files to handle zero values and revised gas fee calculations.
```
Comment (thedarkjester) [inline docs/developers/linea-version/release-notes.mdx:36]: I would be more explicit with something like 
```
to handle messages that are contract calls with zero ETH value being transferred
```

File: docs/developers/linea-version/release-notes.mdx [hunk 4]
```diff
@@ -17,7 +17,52 @@ Find out all you need to know about the latest Linea versions.
...
+
+## Linea Alpha v0.2.1 (June 15, 2023)
+
+### Summary
+
+This release reduces the gas cost of submitting a batch of transactions on L1 by optimizing calldata that is sent by L2 to L1 logs.
```
Comment (thedarkjester) [inline docs/developers/linea-version/release-notes.mdx:42]: consider something like

`.. optimizing L2 logs in the calldata that is sent to L1`

File: docs/developers/linea-version/release-notes.mdx [hunk 5]
```diff
@@ -17,7 +17,52 @@ Find out all you need to know about the latest Linea versions.
...
+  //			address indexed _to,
+  //			uint256 _fee,
+  //			uint256 _value,
+  //			uint256 _salt,
+  //			bytes _calldata,
+  //			bytes32 _messageHash
+  //	    );
+  ```
+
+- **New calldata for L2 originated messages**:
```
Comment (thedarkjester) [inline docs/developers/linea-version/release-notes.mdx:62]: events vs. messages (not critical to change it though)

File: docs/developers/linea-version/release-notes.mdx (hunk 1 above)
Comment (dfox-consensys) [reply docs/developers/linea-version/release-notes.mdx:26]: Date looks wrong?

Comment (cedrabner) [reply docs/developers/linea-version/release-notes.mdx]: @dfox-consensys got the dates based on what's on the notion page so i think they should be okay.

Comment (cedrabner) [reply docs/developers/linea-version/release-notes.mdx]: fixed

File: docs/developers/linea-version/release-notes.mdx (hunk 2 above)
Comment (cedrabner) [reply docs/developers/linea-version/release-notes.mdx]: fixed

File: docs/developers/linea-version/release-notes.mdx (hunk 3 above)
Comment (cedrabner) [reply docs/developers/linea-version/release-notes.mdx]: fixed

File: docs/developers/linea-version/release-notes.mdx (hunk 5 above)
Comment (cedrabner) [reply docs/developers/linea-version/release-notes.mdx]: fixed

File: docs/developers/linea-version/release-notes.mdx (hunk 4 above)
Comment (cedrabner) [reply docs/developers/linea-version/release-notes.mdx]: fixed

**Review by Ludcour — APPROVED** [review]

**Pull request merged**